cmd/wzprof/main.go

```diff
@@ -85,7 +85,8 @@ func (prog *program) run(ctx context.Context) error {
 
 	symbols, err := wzprof.BuildDwarfSymbolizer(compiledModule)
 	if err != nil {
-		return fmt.Errorf("symbolizing wasm module: %w", err)
+		symbols = nil
```

Comment:
Isn't this going to crash when we generate the profile if the symbolizer is nil?

Reply:
Ah nevermind we have a check there https://github.com/stealthrocket/wzprof/blob/main/wzprof.go#L83-L86

What do you think about removing the `nil` check an instead using a no-op implementation of the interface? `nil` as a valid value seems like asking for trouble.

Reply:
Yeah I thought about it, but figured I'll revamp all this when I'll add pclntab support and need to do some kind of symbolizer selection.